docs/upgrade/UPGRADE-v8.0.0.md

```diff
@@ -70,6 +70,8 @@ There you can find links to upgrade notes for other versions too.
             + use FOS\CKEditorBundle\Form\Type\CKEditorType;
             ```
 - update the minimal PHP version in your `composer.json` in `require` and `config.platform` section to `7.2` because version `7.1` is no longer supported in Shopsys Framework ([#1066](https://github.com/shopsys/shopsys/pull/1066))
+- follow instructions in [the separate article](upgrade-instructions-for-backend-api.md) to introduce backend API into your project ([#1055](https://github.com/shopsys/shopsys/pull/1055))
```

Comment:
I would put an emphasis that it is experimental and they may choose not to integrate it to their project yet. eg "If you want to use our experimental API..."

Reply:
Well we decided not to mark upgrade notes "optional", but in this case I agree with you.